index.html

```diff
@@ -21,18 +23,199 @@
+                      <a href=""><img src="assets/angel.svg" alt="Cristian's Angelist" class="social-icon"></a>
+                  </li>
+                  <li class="social-media-item"> 
+                      <a href=""><img src="assets/twitter.svg" alt="Cristian's Twitter" class="social-icon"></a>
+                  </li>
+                  <li class="social-media-item"> 
+                      <a href=""><img src="assets/medium.svg" alt="Cristian's Medium" class="social-icon"></a>
+                  </li>
+              </ul>
+              <!-- Resume button -->
+              <button type="button" class="resume-button">Get my resume</button>
+          </div>
+           <!-- Skills Section -->
+     <ul class="skills-section">
+         <li>
+           <div class="title-container1">
+           <h3 class="skills-title">Languages</h3>
+           <img src="assets/down-arrow.svg" alt="down-arrow-vector" class="skills-image">
+         </div>
+           <ul class="skills-list">
+             <li class="skills-item">
+               <!-- JavaScript -->
+               <div class="language-item">
+                 <figcaption class="language-image-container">
+                   <img class="language-image" src="./assets/js.png" alt="javascript">
+                 </figcaption>
+                 <h2 class="skills-item-title">JavaScript</h2>
+               </div>
+             </li>
+             <li class="skills-item">
+               <!-- HTML -->
+               <div class="language-item">
+                 <figcaption class="language-image-container">
+                   <img src="./assets/html.png" alt="html" class="language-image">
+                 </figcaption>
+                 <h2 class="skills-item-title">HTML</h2>
+               </div>
+             </li>
+             <li class="skills-item">
+               <!-- CSS -->
+               <div class="language-item">
+                 <figcaption class="language-image-container">
+                   <img src="assets/css.png" alt="css" class="language-image">
+                 </figcaption>
+                 <h2 class="skills-item-title">CSS</h2>
+               </div>
+             </li>
+           </ul>
+         </li>
+         <li>
+              <div class="title-container">
+           <h3 class="skills-title">Frameworks</h3>
+           <ul class="skills-container active">
+               <figcaption class="skills-image-container">
+                 <img src="assets/arrow.svg" alt="arrow-vector" class="arrow-image">
+               </figcaption>
+           </ul>
+           </div>
+         </li>
+         <li>
+         <div class="title-container">
+           <h3 class="skills-title">Skills</h3>
+           <ul class="skills-container active">
+             <li>
+               <figcaption class="skills-image-container">
+                 <img src="assets/arrow.svg" alt="arrow-vector" class="arrow-image">
+               </figcaption>
+               </div>
```

Comment:
[OPTIONAL]
- [ ] The `div` in this highlighted line is not properly placed as an enclosing closing tag. kindly fix this for better html structure before merging :+1: :sparkles: